### ISSUE TITLE
problem of net.children

### ISSUE THREAD

**Issue by gss-ucas** [pytorch/pytorch#3065]
Hi, everyone, I have a problem of net.children:
I have got a pretrained model but it has only one child as following:

> BNInception (
    (conv1_7x7_s2): Conv2d(3, 64, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3))
    (conv1_7x7_s2_bn): BatchNorm2d(64, eps=1e-05, momentum=0.1, affine=True)
    (conv1_relu_7x7): ReLU (inplace)
    (pool1_3x3_s2): MaxPool2d (size=(3, 3), stride=(2, 2), dilation=(1, 1))
    (conv2_3x3_reduce): Conv2d(64, 64, kernel_size=(1, 1), stride=(1, 1))
    (conv2_3x3_reduce_bn): BatchNorm2d(64, eps=1e-05, momentum=0.1, affine=True)
    (conv2_relu_3x3_reduce): ReLU (inplace)
    (conv2_3x3): Conv2d(64, 192, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
    (conv2_3x3_bn): BatchNorm2d(192, eps=1e-05, momentum=0.1, affine=True)
    (conv2_relu_3x3): ReLU (inplace)
    (pool2_3x3_s2): MaxPool2d (size=(3, 3), stride=(2, 2), dilation=(1, 1))
    (inception_3a_1x1): Conv2d(192, 64, kernel_size=(1, 1), stride=(1, 1))
    (inception_3a_1x1_bn): BatchNorm2d(64, eps=1e-05, momentum=0.1, affine=True)
    (inception_3a_relu_1x1): ReLU (inplace)
......... 
  )
)

Then, how can I get output of immediate layer in this situation?
please help me, thank you!
